src/kfs.h

```diff
@@ -36,5 +36,6 @@
 #include "lib/includes/kfs_readlink.h"
 #include "lib/includes/kfs_fsync.h"
 #include "lib/includes/kfs_opendir.h"
+#include "lib/includes/kfs_releasedir.h"
```

Comment:
this strange, why not just kfs_releasedir.h?
why lib/includes?

Reply:
it will work without lib/includes, but it clarifies that all included functions build with separate cmakelists as library

Reply:
ok, I'l lcheck